Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3
+
+COPY ./ /workspace/
```

Comment:
You can split this to benefit the caching, requirements are more likely to change less frequently than code.

I ended up with the following Dockerfile:

```
FROM python:alpine
ADD requirements.txt /workspace/requirements.txt
WORKDIR /workspace/
RUN apk add --no-cache libffi openssl && apk add --no-cache libffi-dev openssl-dev build-base && pip install -r requirements.txt && apk del build-base libffi-dev openssl-dev
ADD convert_pipelines.py /workspace/convert_pipelines.py
ENTRYPOINT ["python", "/workspace/convert_pipelines.py"]
```